l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py

```diff
@@ -270,7 +270,7 @@ def getPartnerBase(self, DatiAnagrafici):  # noqa: C901
         if len(partners) > 1:
             for partner in partners:
                 if (
-                    commercial_partner_id
+                    not commercial_partner_id
```

Comment:
Per come l'avevo capito io, a questo punto abbiamo diversi `partners` che sono _equivalenti_ (hanno stessa P.IVA e codice fiscale); vogliamo assicurarci che abbiano tutti lo stesso commercial partner e metterlo in `commercial_partner_id`.

Il loop quindi inizializza `commercial_partner_id` a `False` e inizia a guardarsi ogni `partner`:

1. al primo giro **non** entra in questa condizione perché e imposta `commercial_partner_id = partner.commercial_partner_id.id`
2. nei giri successivi, confronta `partner.commercial_partner_id` con `commercial_partner_id` (il valore preso nel primo giro): se sono diversi c'è inconsistenza, esce, ecc.

Ne abbiamo anche parlato in chiamata, fammi sapere se ti torna

Reply:
Hai ragione.